### ISSUE TITLE
ScrollView with dynamically added Entries are not clickable

### ISSUE THREAD

**Issue by Jake-Derrick** [dotnet/maui#15762]
### Description

When an `Entry` is added dynamicaly with `BindableLayout` in a `StackLayout` in a `ScrollView`, if it increases the height to where the ScrollView must be scrolled the `Entry` cannot be selected.

[screen-capture.webm](https://github.com/dotnet/maui/assets/60721064/3cf98e3b-0630-4221-ae87-e4754e8362ad)

It is possible on the simulator to press tab to select the `Entry`, but it still cannot be selected by a click/press

### Steps to Reproduce

Create a `ScrollView` that contains a `StackLayout` that contains a `BindableLayout.ItemsSource` where the ItemSource is an `ObservableCollection`. Ex:

```
<ScrollView>
    <StackLayout BindableLayout.ItemsSource="{Binding StringList}">
        <BindableLayout.ItemTemplate>
            <DataTemplate>
                <StackLayout HorizontalOptions="FillAndExpand">
                    <Entry Text="{Binding .}" WidthRequest="100"/>
                </StackLayout>
            </DataTemplate>
        </BindableLayout.ItemTemplate>
    </StackLayout>
</ScrollView>
```

See repo for full example

### Link to public reproduction project repository

https://github.com/Jake-Derrick/EntryIssue

### Version with bug

7.0.86

### Last version that worked well

Unknown/Other

### Affected platforms

iOS, I was *not* able test on other platforms

### Affected platform versions

iOS 16

### Did you find any workaround?

CollectionView and ListView will make the Entry selectable, but then you have a nested scrolling

**Comment by PureWeen** [comment]
@Jake-Derrick if you get a chance can you test this on .NET 8? And let us know if it's still an issue.

**Comment by Jake-Derrick** [comment]
> @Jake-Derrick if you get a chance can you test this on .NET 8? And let us know if it's still an issue.

I tried it on version 8.0.0-preview.5.8529, but it had the same issue

**Comment by Jake-Derrick** [comment]
This issue is related to #14624 
A workaround can be found here: https://github.com/dotnet/maui/issues/14624#issuecomment-1538190127

**Comment by diocleziano** [comment]
Good Evening, I have the same problem in "Windows Machine" and with multiple static "Entry" fields.

**Comment by XamlTest** [comment]
Verified this on Visual Studio Enterprise 17.7.0 Preview 4.0. Repro on iOS 16.4 .NET 8, not repro on Windows 11 and Android 13.0-API33 with below Project:
[EntryIssue.zip](https://github.com/dotnet/maui/files/12158236/EntryIssue.zip)

**Comment by hartez** [comment]
#16385 _might_ fix this.

**Comment by PureWeen** [comment]
Can y'all validate this fix on .NET8 preview7?

**Comment by hartez** [comment]
> #16385 _might_ fix this.

Confirmed, this is fixed by #16385.
